.github/workflows/autobuild.yaml

```diff
@@ -120,12 +120,13 @@ jobs:
         "-DLLVM_ENABLE_PLUGINS=OFF"
         ${{ matrix.config.cmake }}
     - name: Ninja
-      run: ninja -C ${{ env.CLANGD_DIR }} clangd
+      run: ninja -C ${{ env.CLANGD_DIR }} clangd clangd-indexer
     - name: Archive
       run: >
         7z a clangd.zip
         ${{ env.CLANGD_DIR }}/LICENSE.TXT
         ${{ env.CLANGD_DIR }}/bin/clangd*
+        ${{ env.CLANGD_DIR }}/bin/clangd-indexer
```

Comment:
drive-by comment:

I believe there would be value in patching those separately. as most of the users is likely to need clangd but not the indexer, and the additional binary would just increase package size.

(also you would want to put a `*` for matchin `.exe` on windows.

Reply:
Good point, thanks!